### ISSUE TITLE
Adjust checkbox design to Nextcloud

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/tasks#506]
We should adjust the checkboxes in Tasks to look like the core Nextcloud ones. :)

Compare:
![checkbox tasks](https://user-images.githubusercontent.com/925062/61304621-bf499580-a7e9-11e9-8e0b-51c9dbcbdfdf.png) ![Checkbox nextcloud](https://user-images.githubusercontent.com/925062/61304619-bf499580-a7e9-11e9-926e-699125f76c27.png)

See https://twitter.com/despens/status/1151098437875699719 by @despens

Documentation for how to do Nextcloud-style checkboxes are at https://docs.nextcloud.com/server/latest/developer_manual/design/html.html#checkboxes-and-radios

cc @nextcloud/tasks @nextcloud/designers

**Comment by raimund-schluessler** [comment]
I don't know, this looks really weird to me:
![Screenshot-2019-7-16 Aufgaben - Nextcloud](https://user-images.githubusercontent.com/2496460/61306486-ff5e4780-a7ec-11e9-8d7f-ecdabc7d6ae5.png)

The checkbox is to small and it doesn't use the Tasks app icon anymore as the checkmark.

**Comment by jancborchardt** [comment]
The size of the checkbox is adjustable, right @skjnldsv @juliushaertl?

And then we can adjust the Tasks app icon too. :)

**Comment by raimund-schluessler** [comment]
> And then we can adjust the Tasks app icon too, being a checkmark in a box. I can do that. :)

My argument with the same icon does not work in this direction 😉 I don't think we should change the app icon. The icon is
- clear, simple and friendly
- unambiguously
- used for many years

If you really think we have to adjust the checkboxes, I can live with it. I just wonder if the nextcloud design checkboxes might be understood as selecting a task rather than completing it.

**Comment by despens** [comment]
I'd appreciate if the look of checkboxes was unified across Nextcloud apps. 

Users see dozens of differently styled checkboxes all the time, in different desktop and web apps, not even to mention mobile apps. If it looks remotely like a checkbox they will assume it is a checkbox. :wink: 

Also, there are tons of apps that use checkboxes for completing tasks, so for usability the exact styling is not an issue. But it would just be nice if at least within Nextcloud the visual clutter could be reduced and user interface elements provide the same functionality also look the same.

**Comment by juliushaertl** [comment]
I agree with @jancborchardt that the checkbox doesn't really fit into the rest of the UI, but also that the primary colored checkbox is more about something is selected then something is done.

How about this:
![image](https://user-images.githubusercontent.com/3404133/61323615-bddf9380-a810-11e9-8569-b098f96ef44e.png)

**Comment by raimund-schluessler** [comment]
> 
> 
> I agree with @jancborchardt that the checkbox doesn't really fit into the rest of the UI, but also that the primary colored checkbox is more about something is selected then something is done.
> 
> How about this:
> ![image](https://user-images.githubusercontent.com/3404133/61323615-bddf9380-a810-11e9-8569-b098f96ef44e.png)

I would like this better than the primary colored checkbox, but this doesn't help in unifying the checkbox appearance all over Nextcloud.

**Comment by jancborchardt** [comment]
Cool, @juliushaertl's suggestion is nice as well. I do think unification of the form is already a good step, it doesn't all need to be primary color.

**Comment by raimund-schluessler** [comment]
Let’s do it like @juliushaertl suggested it then.

**Comment by raimund-schluessler** [comment]
Anyone up for sending a PR? 😉

**Comment by raimund-schluessler** [comment]
@juliushaertl Since we agreed on your design proposal, could you share the CSS for this (in case you have it). PR to adjust it is in https://github.com/nextcloud/tasks/pull/572.

**Comment by juliushaertl** [comment]
Not sure if the PR is actually still needed, it already looks quite fine for me on master. Just a small patch for some spacing issues:

```diff
diff --git a/css/src/style.scss b/css/src/style.scss
index 841a37f..dc9f06a 100644
--- a/css/src/style.scss
+++ b/css/src/style.scss
@@ -1199,9 +1199,14 @@
 		border-radius: 3px;
 		border: 1px solid var(--color-main-text);
 		float: left;
-		margin: 9px;
+		padding: 9px;
+		margin: 7px 7px;
 		opacity: 0.5;
 
+		&.icon-checkmark {
+			margin-top: 10px;
+		}
+
 		&:not(.icon-checkmark) {
 			background: unset;
 		}

```

![image](https://user-images.githubusercontent.com/3404133/63966465-a3e5d180-ca9b-11e9-91b0-7244789c9772.png)

**Comment by juliushaertl** [comment]
Ah sorry, forgot to build the svg sprite. If you just remove the custom background image from the icon-checkmark class and use the server shipped one, it should already be fine. cc @jakobroehrl

**Comment by raimund-schluessler** [comment]
Have a look at #572.

**Comment by raimund-schluessler** [comment]
Fixed with #572.
